server/src/main/java/org/apache/iotdb/db/service/DataNodeServerCommandLine.java

```diff
@@ -158,8 +154,10 @@ private void removeNodesFromCluster(String[] args) throws BadNodeUrlException {
     try (ConfigNodeClient configNodeClient = new ConfigNodeClient()) {
       TDataNodeRemoveResp removeResp = configNodeClient.removeDataNode(removeReq);
       logger.info("Remove result {} ", removeResp.toString());
-    } catch (TException e) {
-      logger.error("send remove Data Node request failed!", e);
+      if (removeResp.getStatus().getCode() != TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
+        throw new IoTDBException(
+            removeResp.getStatus().toString(), removeResp.getStatus().getCode());
```

Comment:
If it's better to set the error message `send remove Data Node request failed!` in IoTDBException

Reply:
ConfigNode returned status has more detail message,  In my test case, it is :

org.apache.iotdb.commons.exception.IoTDBException: TSStatus(code:916, message:lack replication, allow most removed Data Node size : 0)
	at org.apache.iotdb.db.service.DataNodeServerCommandLine.removeNodesFromCluster(DataNodeServerCommandLine.java:224)